python/ray/tune/search/sample.py

```diff
@@ -421,17 +421,34 @@ def sample(
                 random_state = _BackwardsCompatibleNumpyRng(random_state)
             # do not use .choice() directly on domain.categories
             # as that will coerce them to a single dtype
-            indices = random_state.choice(
+            return random_state.choice(
```

Comment:
We should also make sure to document the change in behavior from sampling the values to sampling indices.

Reply:
n/a at this point